Resources/Prototypes/_RMC14/Entities/Clothing/Belt/belts.yml

```diff
@@ -48,6 +48,36 @@
       - CMMagazineSmg
   - type: FixedItemSizeStorage
 
+- type: entity
+  parent: CMBeltBaseStorage
+  id: RMCM276ShotgunShellLoadingRig
+  name: M276 pattern shotgun shell loading rig
+  description: An ammunition belt designed to hold shotgun shells or individual bullets.
+  components:
+  - type: Sprite
+    sprite: _RMC14/Objects/Clothing/Belt/shotgun.rsi
+  - type: Clothing
+    sprite: _RMC14/Objects/Clothing/Belt/shotgun.rsi
+  - type: Item
+    size: Large
+  - type: Storage
+    maxItemSize: Large
+    grid:
+    - 0,0,5,1 # 3 slots
+    whitelist:
+      tags:
+      - RMCBoxShotgunShells
+#      - # TODO RMC14 shotgun shell handfuls
+
+      components:
+      - BallisticAmmoProvider
```

Comment:
This will let anything with a BallisticAmmoProvider component in I think, you want just the tag filter
```suggestion
    whitelist:
      tags:
      - RMCBoxShotgunShells
#      - # TODO RMC14 shotgun shell handfuls
```